packages/terra-avatar/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "terra-avatar",
+  "main": "lib/Avatar.js",
+  "version": "0.0.0",
+  "description": "{insert description}",
```

Comment:
Add description here `The terra-avatar component displays an avatar which can be either an image or text. If neither an image or text is provided, a variant-based fallback icon is used. The avatar displays in a circular wrapper.`

Reply:
Updated in this [commit](https://github.com/cerner/terra-core/commit/855d5fc5c14ce5dc2a29cb07d7d5575af2320962).